### PR TITLE
Seamus Tasks (Elevation, Buttons, Spacing)

### PR DESCRIPTION
**Pull request by sloan-seamus**
Fixed the elevation issues (?) in all pages.
	- All pages now start with elevation 4
	- Elements with elements on top now use the the elevation + 2 (i.e. file table in edit claims is on z6)

Fixed buttons on navigation toolbar
	- Buttons now blend in with the toolbar

Fixed spacing in edit claims
	- File section now looks similar to the other sections in terms of spacing

Additional:
	- Removed the icons from the arrow/delete buttons in edit claim (Text is clearer)

### CODE REVIEW

**Comment by sloan-seamus** [comment]
@ej3076 pls review me and merge me

File: frontend/insurance/src/app/shared/components/edit-claim/edit-claim.component.ts [hunk 1]
```diff
@@ -160,7 +160,7 @@ export class EditClaimSharedComponent implements OnInit {
         name: this.insurerForm.controls['insurerName'].value
           ? this.insurerForm.controls['insurerName'].value
           : this.claim.insurer.name,
-        has_gap: this.insurerForm.controls['gap'].value
+        has_gap: Boolean(this.insurerForm.controls['gap'].value)
```
Comment (ej3076) [inline frontend/insurance/src/app/shared/components/edit-claim/edit-claim.component.ts:163]: You're mapping a string to a boolean here, and that string is either a "Yes" or "No", right? You should probably just switch this to a radio button of true or false.

**Review by ej3076 — CHANGES_REQUESTED** [review]
Changes

File: frontend/insurance/src/app/shared/components/edit-claim/edit-claim.component.ts (hunk 1 above)
Comment (dsifford) [reply frontend/insurance/src/app/shared/components/edit-claim/edit-claim.component.ts:163]: or just use the correct property of checkboxes... You don't look up value, you look up `checked`. That's the boolean

Comment (sloan-seamus) [reply frontend/insurance/src/app/shared/components/edit-claim/edit-claim.component.ts:163]: That was the plan but the way you made all these forms makes no sense compared to the way we created the form for New-Claim-Component.

**Comment by ej3076** [comment]
Merging - @sloan-seamus will be making more changes in his next branch.

**Review by ej3076 — APPROVED** [review]
Approved based on changes that will be made tomorrow.

**Pull request merged**